packages/firestore/test/unit/specs/recovery_spec.test.ts

```diff
@@ -34,11 +38,7 @@ describeSpec('Persistence Recovery', ['no-ios', 'no-android'], () => {
           .client(1)
           .expectPrimaryState(false)
           .userSets('collection/a', { v: 1 })
-          .failDatabaseTransactions({
-            'Locally write mutations': true,
-            'Synchronize last document change read time': true,
-            'Lookup mutation documents': true
-          })
+          .failDatabaseTransactions('Lookup mutation documents')
```

Comment:
This seems like a different test than what was here before. Previously the write of the mutation to IndexedDB would fail (as would the lookup) but now this is failing just the lookup and allowing the write to succeed?

Alternatively, if this is working as intended, better names would help make it clearer just which transaction is failing. For example, if this were named `multitab.secondary.mutationLookup` it would be much, much clearer.

Reply:
There is already a dedicated test that test if "Locally write mutations" fails ("Recovers when write cannot be persisted"). The fact that "Locally write mutations" is part of this list seems like an oversight to me - the mutation gets written right before and not after we start injecting failures.

There are certainly improvements to the transaction names that we should make before we port to Android/iOS, but I would like to point out that 'Lookup mutation documents' is also used by the primary tab to load a mutation that has been written by a secondary tab.